website/docs/explorer/operations.mdx

```diff
@@ -225,6 +225,52 @@ indexer_inflight_requests{action="Add"} 0
 indexer_inflight_requests{action="Remove"} 0
 ```
 
+#### Management
+
+Explorer management path is composed of two components exporting metrics:
+
+- Objects Cleaner Status
+- Objects Cleaner Remove Objects Requests
+
+The following metrics are available to monitor its health.
```

Comment:
this line might be better earlier as when you first introduce the metrics.

Reply:
Changed it to:

```
#### Management

Explorer management contains the 'Objects Cleaner` component exporting metrics. The following metrics are available to monitor its health:

- Objects Cleaner Status
- Objects Cleaner Remove Objects Requests

##### Objects Cleaner Status
```